docs/pages/pmd/languages/java_versions.md

```diff
@@ -1,39 +0,0 @@
----
-title: Supported Java Versions
-permalink: pmd_languages_java_versions.html
----
-
-## Overview of supported Java language versions
```

Comment:
I think this section should be added to `java.md` and not thrown away. The cli help lists supported versions but does not provide historical information.

Reply:
This [is already part of `java.md`](https://github.com/pmd/pmd/blob/76c27d41723a5b247a26d1849f4e2a6318ac92a2/docs/pages/pmd/languages/java.md?plain=1#L12-L46)

This was a verbatim copy with nothing to add.

Reply:
good catch!